client/landing/gutenboarding/onboarding-block/plans/index.tsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useRef, useState } from 'react';
+import { useI18n } from '@automattic/react-i18n';
+import { Button } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import PlansGrid from '../../components/plans/plans-grid';
+import SignupForm from '../../components/signup-form';
+import Link from '../../components/link';
+import { useSelectedPlan } from '../../hooks/use-selected-plan';
+import { useTrackStep } from '../../hooks/use-track-step';
+import { useFreeDomainSuggestion } from '../../hooks/use-free-domain-suggestion';
+import { STORE_KEY as PLANS_STORE } from '../../stores/plans';
+import { USER_STORE } from '../../stores/user';
+import { STORE_KEY as ONBOARD_STORE } from '../../stores/onboard';
+import { usePath, Step } from '../../path';
+
+import './style.scss';
+
+export default function PlansStep() {
+	const { __ } = useI18n();
+	const makePath = usePath();
+
+	const [ showSignupDialog, setShowSignupDialog ] = useState( false );
+	const currentUser = useSelect( ( select ) => select( USER_STORE ).getCurrentUser() );
+
+	const { createSite } = useDispatch( ONBOARD_STORE );
+
+	const { setPlan } = useDispatch( PLANS_STORE );
+	const plan = useSelectedPlan();
+
+	const freeDomainSuggestion = useFreeDomainSuggestion();
+
+	// Keep a copy of the selected plan locally so it's available when the component is unmounting
+	const selectedPlanRef = useRef();
+	useEffect( () => {
+		selectedPlanRef.current = plan?.getStoreSlug();
+	}, [ plan ] );
+
+	useTrackStep( 'Plans', () => ( {
+		selected_plan: selectedPlanRef.current,
+	} ) );
+
+	const handleCreateSite = ( username: string, bearerToken?: string ) => {
+		createSite( username, freeDomainSuggestion, bearerToken, plan.getStoreSlug() );
+	};
+
+	return (
+		<>
+			<PlansGrid
+				renderConfirmButton={ ( planSlug ) => (
+					<Button
+						isPrimary
+						onClick={ () => {
+							setPlan( planSlug );
+							currentUser ? handleCreateSite( currentUser.username ) : setShowSignupDialog( true );
+						} }
+					>
+						{ __( 'Continue' ) }
+					</Button>
+				) }
+				onPlanChange={ setPlan }
+				selectedPlanSlug={ plan.getStoreSlug() }
+				cancelButton={
+					<Link className="plans__back-link" isLink to={ makePath( Step.Style ) }>
+						{ __( 'Go back' ) }
+					</Link>
```

Comment:
Styling of the action buttons is a bit different than the one used for modal.

Plans Modal:
<img width="350" alt="Screenshot 2020-05-11 at 10 13 48" src="https://user-images.githubusercontent.com/14192054/81542221-fabadf00-937c-11ea-92c6-3a2ef86e6079.png">
<img width="348" alt="Screenshot 2020-05-11 at 10 13 36" src="https://user-images.githubusercontent.com/14192054/81542222-fbec0c00-937c-11ea-8ccb-7b80d279ec12.png">

Plans Page: